stories/index.js

```diff
@@ -16,8 +19,10 @@ import { Table } from '../src/components/Table';
 import TableContainer from '../src/components/TableContainer';
 import ColumnDefinition from '../src/components/ColumnDefinition';
 import RowDefinition from '../src/components/RowDefinition';
+const { SettingsWrapper, SettingsToggle, Settings } = '../src/components';
 import _ from 'lodash';
-import { columnIdsSelector, stylesForComponentSelector } from '../src/selectors/dataSelectors';
+import * as actions from '../src/actions';
+import * as selectors from '../src/selectors/dataSelectors';
```

Comment:
In retrospect, I should have merged. Regardless, the conflicts were overlapping import lists here. Rather than merge the lists, it seems reasonable to import the full namespace. This will change again with the import refactoring in #618.